packages/forma-36-react-components/src/components/TextInput/TextInput.tsx

```diff
@@ -20,6 +20,7 @@ export interface TextInputPropTypes {
   onBlur?: (...args: any[]) => any;
   onCopy?: (...args: any[]) => any;
   value?: string;
+  inputRef?: any;
```

Comment:
@suevalov I'm not sure what this type should be. Setting it to `any` isn't ideal, but also makes TypeScript upset in the Storybook example:

```
Property 'focus' does not exist on type '{}'.
```

Also, maybe this isn't the right way to handle refs, just seemed the simplest for most use cases.

Reply:
I believe it should be `RefObject<HTMLInputElement>`

Reply:
`import React, { RefObject } from 'react';`

Reply:
Ahh, I also needed to update how I created the ref in the story.

```
const textInputRef = React.createRef<HTMLInputElement>();
```